src/stservo/gui/servo_gui.py

```
@@ -1544,6 +1701,280 @@ def clear_sync_results(self):
+                defaultextension=".txt",
+                filetypes=[("Text files", "*.txt"), ("All files", "*.*")],
+                title="Export Register Values"
+            )
+            
+            if not filename:
+                return
+            
+            timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+            
+            with open(filename, 'w') as f:
+                f.write(f"STServo Register Export\n")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)